.github/issue_template.md

```diff
@@ -0,0 +1,18 @@
+# Disclaimer
+If your Mafia or TourGuide version is wildy out of date when posting this issue, it may already be resolved and your issue may be closed. Please keep your things up to date!
+
+# Basic Data
+
+* KolMafia version:
+* TourGuide commit (`ash git_info("tourguide")`):
```

Comment:
i think i would prefer if they reported tourguide version, located in the bottom of tourguide, because i'm not entirely sure how easy it will be for us to track down how recent their version is. the one unfortunate part is that this does mean we need to be clearer on pushing the versioning (located here) but i think that is more human readable. https://github.com/loathers/TourGuide/blob/09e0658b2b99da7549daad031780a74dbd49cfa7/Source/relay/TourGuide/Settings.ash#L2

Reply:
The commit number is very long, but you can search the repo for the commit number and pull up the exact PR immediately.